windows/src/global/inc/keyman64.h

```diff
@@ -141,148 +142,158 @@
 
 typedef struct tagSTORE
 {
-	DWORD dwSystemID;
-	PWSTR dpName;
-	PWSTR dpString;
-} STORE, *LPSTORE;
+  DWORD dwSystemID;
+  PWSTR dpName;
+  PWSTR dpString;
+} STORE, * LPSTORE;
```

Comment:
Ahh the whitespace changes have happened again will come back and fix this.  There plans to bigger updates to this in a seperate PR. So will pull them in then.